package.json

```diff
@@ -26,7 +26,7 @@
     "uglify": "browserify -g uglifyify ./assets/js/index.js > ./public/js/bundle.js",
     "browserify": "browserify ./assets/js/index.js > ./public/js/bundle.js",
     "create:public": "mkdir -p ./public/css ./public/images ./public/js",
-    "postinstall": "npm run create:public; npm run sass; npm run copy:images; npm run uglify"
+    "postinstall": "npm run create:public && npm run sass && npm run copy:images && npm run uglify"
```

Comment:
Much better `bash`ing, but I would prefer something of this complexity to be handled by a task runner like `grunt` or `gulp`